_includes/search.html

```diff
@@ -1,6 +1,6 @@
 <div id="search" class="menu-target">
   <input id="indicator_search" title="{{ t.search.indicator_search }}" placeholder="{{ t.search.indicator_search }}" data-url="{{ site.baseurl }}{{ baseurl_folder }}/indicators.json" data-pageurl="{{ site.baseurl }}{{ baseurl_folder }}/search/?q=" />
   <label for="indicator_search">
-    <button id="search-btn" type="submit"><i class="fa fa-search" aria-hidden="true"></i></button><span>{{ t.search.search }}:</span>
+    <button id="search-btn" type="submit"><i class="fa fa-search" aria-hidden="false" aria-label="Indicator search"></i></button><span>{{ t.search.search }}:</span>
```

Comment:
Since it's an aria-label I suspect it needs to be translated? Luckily we have this as a translation key already: 
`{{ t.search.indicator_search }}`